README.md

```diff
@@ -173,31 +188,54 @@ You should then have a symlinked directory at `~/openmrs/[serverId]/configuratio
 
 ### Step 6: Start up the server
 
+#### Using the Invoke file
 ```
-$ mvn openmrs-sdk:run -DserverId=[serverId]
+$ invoke run -sk
 ```
 
 It should run for several minutes, setting up the database, (you may have to go to http://localhost:8080/openmrs to trigger this) BUT, in the end, it will fail.  You should cancel the current run (Ctrl-C in the terminal window).
 
-After it fails, notice that a openmrs-runtime.properties file should have been created in the ~/openmrs/[serverId]
+```
+$ invoke configure
+```
+And check that the utput looks okay. If so,
+```
+invoke run -sk
+```
+once again. Startup should take several minutes as it loads in all 
+required metadata, etc, for the first time.
+
+Future runs can be executed with `invoke run`, which automatically
+does a git pull, maven deploy, and enables all modules before running.
+
```

Comment:
Thanks for adding information about this in, seems like a helpful tool.  I had written my own bash-based sdk helper tool years ago.  Python is definitely more powerful and user friendly.

My inclination with this commit is to not try to intertwine the invoke stuff with the rest of the docs, but to have the main docs go through the full steps "the hard way", explaining at each stage what is happening and why.  Then, have a section on the invoke tool which goes through it's features in detail and describes what commands are available for particular things.

Reply:
Makes sense. Thanks Mike